### DISCUSSION TITLE
Tension/compression/Torsion deformation and Crack propagation

### DISCUSSION THREAD

**Discussion by dikshanshs** [neperfepx/FEPX#37] [Q&A] [answered]
@rquey @acmelab-ua 
Is there any feature for compressive and torsional deformation of microstructures? 
How to initiate crack propagation?

**Comment by mkasemer** [comment, marked as answer]
Hi @dikshanshs,

Currently, there is no feature for dual tension/torsion of microstructures. However, you could write your own custom boundary conditions for the uniaxial driver to consider velocity vectors that are not aligned with the sample axis. You could create a set of vectors for the nodes on that surface that effectively apply both a axial velocity (either tension or compression) and a torsional velocity.

FEPX does not consider any fracture models.
